mod_reforged/msu_systems/mod_settings.nut

```diff
@@ -13,3 +13,10 @@ tacticalTooltipPage.addRangeSetting("CollapseActivesWhenX", 5, 0, 20, 1, "Collap
 tacticalTooltipPage.addBooleanSetting("TacticalTooltip_CollapseAsText", false, "Collapse as Text", "If enabled, then skills collapse using their names as text, otherwise they collapse using their icons.");
 tacticalTooltipPage.addBooleanSetting("ShowStatusPerkAndEffect", true, "Show Status Perk And Effect", "Some Perks are also Status Effects. Usually their Effect is hidden until some condition is fulfilled. When this setting is enabled, these perks show up in the Perks category even when they show up under Effects (e.g. when their effect is active). When disabled, when they appear under Effects, they will be hidden from the Perks category. This can help save space on the tooltip.");
 tacticalTooltipPage.addBooleanSetting("HeaderForEmptyCategories", false, "Show Header for empty categories");
+
+::Reforged.Mod.Keybinds.addSQKeybind("WaitRound", "ctrl+space", ::MSU.Key.State.Tactical, function()
+{
+	if (this.m.MenuStack.hasBacksteps() || this.isInputLocked() || this.isInCharacterScreen()) return;
```

Comment:
Newline the return. I'd do it without braces in this case to keep things concise.

Reply:
This is how MSU does the formatting of keybind conditions. 
I copied it thinking that in this case you are ok with it considering you must have approved it over there too

Reply:
Here is the link to that section in MSU https://github.com/MSUTeam/MSU/blob/e0f6edf0465ee0f46c49d199009acd2bcc462ece/msu/vanilla_mod/vanilla_keybinds.nut#L287

I'm really interested in whether you think that section in MSU is hard to read/parse and should be changed.

Reply:
That's older MSU code before Enduriel decided that we need to newline after `if` conditions. Older MSU code has many single-line `if` statements. I don't really mind the way it looks in this case in a single line but I was thinking that Enduriel would maybe now want it to be newlined and I like newlined too (without braces in this case).

Reply:
This should still `return false`. I just realized this isn't something we've really documented but the `return null` is handled to preserve backwards compatibility, not because it's the 'right' way to do this.